doc/src/sdk-user-manual-noninteractive.qdoc

```diff
@@ -67,8 +67,8 @@ maintenance.
         \endlist
 \endtable
 
-Execution on a headless system is possible with the help of the standard Qt
-option \c {-platform minimal}, likely in combination with \c --verbose.
+Execution on a headless system is possible after setting \c QT_QPA_PLATFORM=minimal in the
+environment, likely in combination with the \c --verbose option to the installer.
```

Comment:
I'd prefer using the term "environment variable"

Reply:
There is not only the variable name but also the specific value mentioned, so I preferted this formulation over "after setting the environment variable \c QT_QPA_PLATFORM to \c minimal" or similar. One cannot simply copy & paste that :)

Reply:
Just add the variable to the examples section below and the problem is solved. So in this section, say
`Execution on a headless system is possible after setting the \c QT_QPA_PLATFORM environment variable to value "minimal", likely in combination with the \c --verbose option to the installer. ` and then in the examples section
`$ QT_QPA_PLATFORM=minimal ./installer.run --verbose non-interactive=1 accept-licenses=1`

Reply:
That would hurt readability. Updated.